src/gstspotifysrc.c

```diff
@@ -198,20 +172,20 @@ gst_spotify_src_class_init (GstSpotifySrcClass * klass)
   gobject_class->get_property = gst_spotify_src_get_property;
 
   g_object_class_install_property (gobject_class, PROP_USER,
-      g_param_spec_string ("user", "Username", "Username for premium spotify account",
+      g_param_spec_string ("user", "Username", "Username for premium Spotify account",
           DEFAULT_PROP_USER, G_PARAM_READWRITE));
 
   g_object_class_install_property (gobject_class, PROP_PASS,
-      g_param_spec_string ("pass", "Password", "Password for premium spotify account",
-	  DEFAULT_PROP_PASS, G_PARAM_READWRITE));
+      g_param_spec_string ("pass", "Password", "Password for premium Spotify account",
+	      DEFAULT_PROP_PASS, G_PARAM_READWRITE));
 
   g_object_class_install_property (gobject_class, PROP_URI,
-      g_param_spec_string ("uri", "URI", "A URI", DEFAULT_PROP_URI,
-          G_PARAM_READWRITE));
+      g_param_spec_string ("uri", "URI", "A Spotify track URI",
```

Comment:
Yes, a track. I checked, it then uses sp_link_as_track on a result from sp_link_create_from_string (as opposed to using sp_link_create_from_playlist). So, it is a Spotify track URI.